lib/HTFeed/PackageType/Simple/Download.pm

```diff
@@ -0,0 +1,73 @@
+package HTFeed::PackageType::Simple::Download;
+
+use warnings;
+use strict;
+use base qw(HTFeed::Stage::Download);
+use HTFeed::Config qw(get_config);
+use Log::Log4perl qw(get_logger);
+
+my $download_base = "dropbox:";
+
+sub run {
+  my $self = shift;
+
+  my $volume = $self->{volume};
+  # check if the item already exists at $volume->get_sip_location()
+  my $sip_loc = $volume->get_sip_location();
+  if (-f $sip_loc) {
+    get_logger->trace("$sip_loc already exists");
+  } else {
+    my $rclone_config_path = get_config('rclone_config_path');
```

Comment:
Probably worth extracting this block to a separate method, esp. since it isn't really feasible to have the `if (-f $sip_loc)` as a `return if ... ` -style guard clause (since we still have to do the stuff at the end)